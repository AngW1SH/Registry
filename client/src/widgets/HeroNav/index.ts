import HeroNav from "./ui/HeroNav";

export { HeroNav };
