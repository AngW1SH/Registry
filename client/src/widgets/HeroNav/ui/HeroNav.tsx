import { ButtonWithIcon } from "@/shared/ui";
import { FC } from "react";

interface HeroNavProps {}

const HeroNav: FC<HeroNavProps> = () => {
  return (
    <ul className="flex w-full flex-col justify-between gap-3 sm:flex-row">
      <ButtonWithIcon
        icon="/calendar-icon-black.svg"
        className="w-full gap-1 bg-white-rgba px-2 py-2 text-sm sm:py-4 lg:gap-5 lg:p-5 lg:text-base xl:text-lg"
      >
        Календарь проектов
      </ButtonWithIcon>
      <ButtonWithIcon
        icon="/offer-icon-black.svg"
        className="w-full gap-1 bg-white-rgba px-2 py-2 text-sm sm:py-4 lg:gap-5 lg:p-5 lg:text-base xl:text-lg"
      >
        Вакансии
      </ButtonWithIcon>
      <ButtonWithIcon
        icon="/filters-icon-black.svg"
        className="w-full gap-1 bg-secondary-rgba px-2 py-2 text-sm sm:py-4 lg:gap-5 lg:p-5 lg:text-base xl:text-lg"
      >
        Расширенный поиск
      </ButtonWithIcon>
    </ul>
  );
};

export default HeroNav;
