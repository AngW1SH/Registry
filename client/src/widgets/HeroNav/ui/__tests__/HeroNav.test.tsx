import { render } from "@testing-library/react";
import HeroNav from "../HeroNav";

describe("HeroNav Widget UI", () => {
  test("snapshot test", () => {
    const { asFragment } = render(<HeroNav />);

    expect(asFragment()).toMatchSnapshot();
  });
});
