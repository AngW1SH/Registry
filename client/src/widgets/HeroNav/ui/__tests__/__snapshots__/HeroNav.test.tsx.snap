// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeroNav Widget UI snapshot test 1`] = `
<DocumentFragment>
  <ul
    class="flex w-full flex-col justify-between gap-3 sm:flex-row"
  >
    <button
      class="flex items-center justify-center rounded-2xl bg-secondary-rgba w-full gap-1 bg-white-rgba px-2 py-2 text-sm sm:py-4 lg:gap-5 lg:p-5 lg:text-base xl:text-lg"
    >
      <div
        class="relative flex h-8  w-8 items-center justify-center"
      >
        <img
          alt=""
          data-nimg="fill"
          decoding="async"
          loading="lazy"
          src="/calendar-icon-black.svg"
          style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
        />
      </div>
      <span
        class="font-semibold"
      >
        Календарь проектов
      </span>
    </button>
    <button
      class="flex items-center justify-center rounded-2xl bg-secondary-rgba w-full gap-1 bg-white-rgba px-2 py-2 text-sm sm:py-4 lg:gap-5 lg:p-5 lg:text-base xl:text-lg"
    >
      <div
        class="relative flex h-8  w-8 items-center justify-center"
      >
        <img
          alt=""
          data-nimg="fill"
          decoding="async"
          loading="lazy"
          src="/offer-icon-black.svg"
          style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
        />
      </div>
      <span
        class="font-semibold"
      >
        Вакансии
      </span>
    </button>
    <button
      class="flex items-center justify-center rounded-2xl bg-secondary-rgba w-full gap-1 bg-secondary-rgba px-2 py-2 text-sm sm:py-4 lg:gap-5 lg:p-5 lg:text-base xl:text-lg"
    >
      <div
        class="relative flex h-8  w-8 items-center justify-center"
      >
        <img
          alt=""
          data-nimg="fill"
          decoding="async"
          loading="lazy"
          src="/filters-icon-black.svg"
          style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
        />
      </div>
      <span
        class="font-semibold"
      >
        Расширенный поиск
      </span>
    </button>
  </ul>
</DocumentFragment>
`;
